source/_integrations/nanoleaf.markdown

```diff
@@ -50,3 +51,58 @@ name:
 4. The output should include the auth token like *{"auth_token":"xxxxxxxxxxxxxxxxxxxxx"}*, copy the resulting token into your configuration
 
 If you get a 403 Forbidden message, you probably did not press the *ON* button long enough. The time-frame to get a valid token is only 30 seconds, so you have to be quick to issue the curl request.
+
+### Getting useful values for automations
+
+First set up some effects with the Nanoleaf app, and have them installed on the device.
+
+Once installed, go to "Developer Tools" and find the state of the device, eg `light.bedroom_triangles`. You should end up with something like this:
+
+```yaml
+# Example nanoleaf state:
```

Comment:
Dropping states in documentation causes confusion in general, as people try to use them. Please remove this blob.

Reply:
Ok, can do. However when trying to set up my Nanoleaf I had quite a bit of confusing working out how to find the names of the effects that I could use and how to call them, I had to read the integration code just to work out what it was expecting. This is why I have made the examples of how to find the effects from the state and then call the service to set the lights later.

Is there a better way to communicate this? At the moment the page feels like "here's how to create the light entity" but with no info on how to actually use the entity that was created.

Reply:
If you click on a light, it just shows them...

![image](https://user-images.githubusercontent.com/195327/107847623-6f7be780-6ded-11eb-822b-67f3d8a537a0.png)

Reply:
Ok, that's good. Can I put a screenshot in this document? I'm not sure how I can do that.

Reply:
Yes, however, the below concerns are in the same region at this one.

It is general Home Assistant functionality, and not specific to this integration.